django/core/mail/message.py

```diff
@@ -258,6 +258,8 @@ def get_connection(self, fail_silently=False):
         return self.connection
 
     def message(self):
+        HEADER_WITH_ONE_OCCURRENCE = ["to", "reply-to", "cc"]
+
```

Comment:
This shouldn't be defined as a variable in the method.
Could we move it and make it as a class attribute instead?

And we make probably make it a tuple :)